README.adoc

```diff
@@ -3435,10 +3435,10 @@ at the very end of the boot logs.
 
 === Init environment
 
-Documented at https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html[]:
```

Comment:
BTW, never ever use latest on a link. It WILL break one day. I'll fix this later on with my commit.

Reply:
Oh alright! I will keep that in mind